from ..exceptions import QuafuError

class UserError(QuafuError):
    pass
